File: ALODWebUtility/PageTitles/PageTitle.vb
Imports ALOD.Data

Namespace PageTitles

    Public Class PageTitle

        Protected _pageId As Integer = 0
        Protected _title As String = String.Empty

        Public Sub New()

        End Sub

        Public Property PageId() As Integer
            Get
                Return _pageId
            End Get
            Set(ByVal value As Integer)
                _pageId = value
            End Set
        End Property

        Public Property Title() As String
            Get
                Return _title
            End Get
            Set(ByVal value As String)
                _title = value
            End Set
        End Property

        Public Sub Insert()
            Dim adapter As New SqlDataStore()
            adapter.ExecuteNonQuery("core_pageTitles_sp_Insert", _title)

        End Sub

        Public Sub ToDataRow(ByRef row As DataSets.PageTitlesRow)
            row.pageId = _pageId
            row.title = _title
        End Sub

    End Class

End Namespace

File: ALODWebUtility.CS/PageTitles/PageTitle.cs
using ALOD.Data;
using ALODWebUtility.DataAccess;


namespace ALODWebUtility.PageTitles
{
    public class PageTitle
    {
        protected int _pageId = 0;
        protected string _title = string.Empty;

        public PageTitle()
        {
        }

        public int PageId
        {
            get
            {
                return _pageId;
            }
            set
            {
                _pageId = value;
            }
        }

        public string Title
        {
            get
            {
                return _title;
            }
            set
            {
                _title = value;
            }
        }

        public void Insert()
        {
            SqlDataStore adapter = new SqlDataStore();
            adapter.ExecuteNonQuery("core_pageTitles_sp_Insert", _title);
        }

        public void ToDataRow(ref DataSets.PageTitlesRow row)
        {
            row.pageId = _pageId;
            row.title = _title;
        }
    }
}
